src/components/AddNewTodoModal.tsx

```diff
@@ -62,12 +62,12 @@ export function AddNewTodoModal(props: IAddNewTodoModalProps) {
             renderInput={(props) => <TextField {...props} />}
             label="Expiration  Date"
             ampm={false}
-            inputFormat="dd/MM/yyyy HH:mm"
+            inputFormat="dd.MM.yyyy HH:mm"
             value={expirationDate}
             onChange={(newValue) => onDatePicking(newValue)}
           />
         </LocalizationProvider>
-        <Box className="button-box">
+        <Box sx={{ display: 'flex', justifyContent: 'flex-end' }}>
```

Comment:
again inline styles

Reply:
I removed it on the US002 branch, but this branch is based on an unimproved US002 branch, so it's just some merge problems. I don't do that anymore)
to be fixed